### ISSUE TITLE
Country / stateProvince mismatch for holotype

### ISSUE THREAD

**Issue by dshorthouse** [plazi/community#221]
Country shared is Turkey but should be United States of America.

**Context**
[GBIF occurrence](https://www.gbif.org/occurrence/3071355676)
[Plazi reference](http://treatment.plazi.org/id/D0571FB1486250FCA14C6998C32EA548)

**Comment by jugiora** [comment]
Hi!
The issue was already fixed.
You may just wait for GBIF to update.

**Comment by dshorthouse** [comment]
Thanks for looking into this, but the occurrence record has evaporated at GBIF so I have no way to verify that all is well. I wasn't expecting that for a record about a holotype.

**Comment by flsimoes** [comment]
@dshorthouse we certainly did not expect this as well
We're investigating why it just outright disappeared.

**Comment by gsautter** [comment]
Guess you need to remove the `collectingCountry` annotation from "Turkey" in "Turkey Creek" ... surely a weird edge case for a country name occurring in the name of the specific collecting location.

**Comment by flsimoes** [comment]
> Guess you need to remove the `collectingCountry` annotation from "Turkey" in "Turkey Creek" ... surely a weird edge case for a country name occurring in the name of the specific collecting location.

Would just that cause the whole GBIF record to disappear?

**Comment by gsautter** [comment]
Most likely not ... but it's up to them to delete removed records when they take in a new version of a DwCA ... preciously little we could do in that regard.

**Comment by myrmoteras** [comment]
No. The change of the mc annotation causes normally this

New species of the day<https://tb.plazi.org/GgServer/static/newToday.html>
________________________________

**Comment by gsautter** [comment]
It normally should, definitely.

> No. The change of the mc annotation causes normally this 

It normally should, definitely.

**Comment by dshorthouse** [comment]
> Most likely not ... but it's up to them to delete removed records when they take in a new version of a DwCA ... preciously little we could do in that regard.

AFAIK, if the `datasetKey` and the `occurrenceID` remain the same, then GBIF does not mint a new `gbifID`. If either do, then a `gbifID` evaporates and is not reused, but can be resuscitated if the original `datasetKey` & `occurrenceID` combination is shared anew.

**Comment by gsautter** [comment]
> AFAIK, if the `datasetKey` and the `occurrenceID` remain the same, then GBIF does not mint a new `gbifID`. If either do change from source, then a `gbifID` evaporates and is not reused, but can be resuscitated if the original `datasetKey` & `occurrenceID` combination from the source is shared anew.

Absolutely, but the record should update as attributes change.

On the other hand, considering how recent the correction was made (less than an hour ago), maybe we should simply give the GBIF systems some time to ingest the update.

**Comment by dshorthouse** [comment]
> Absolutely, but the record should update as attributes change.
> 
> On the other hand, considering how recent the correction was made (less than an hour ago), maybe we should simply give the GBIF systems some time to ingest the update.

Indeed, though I noticed it had evaporated 3 days ago after @jugiora's first comment that the issue had been fixed, only just commenting today, having also hoped for ghosts to disperse.

**Comment by flsimoes** [comment]
> > AFAIK, if the `datasetKey` and the `occurrenceID` remain the same, then GBIF does not mint a new `gbifID`. If either do change from source, then a `gbifID` evaporates and is not reused, but can be resuscitated if the original `datasetKey` & `occurrenceID` combination from the source is shared anew.
> 
> Absolutely, but the record should update as attributes change.
> 
> On the other hand, considering how recent the correction was made (less than an hour ago), maybe we should simply give the GBIF systems some time to ingest the update.

The recent update was my edit.
